jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -215,8 +216,8 @@ private enum ProtectedTargetType
     private String[] _vconnectors; // connector portion, matching _vhosts array
     private Logger _logger;
     private boolean _allowNullPathInfo;
-    private int _maxFormKeys = Integer.getInteger(MAX_FORM_KEYS_KEY, DEFAULT_MAX_FORM_KEYS);
-    private int _maxFormContentSize = Integer.getInteger(MAX_FORM_CONTENT_SIZE_KEY, DEFAULT_MAX_FORM_CONTENT_SIZE);
+    private int _maxFormKeys = MAX_FORM_UNSET;
```

Comment:
This change would prevent the use of System properties being used, which would be removal of long released behaviour.  You could just add in support for the context attributes, or we could punt this in favour of a different solution for `jetty-web.xml`.

Reply:
The support for these two System properties was just moved to the doStart() location, not removed.

Reply:
I see it now in `doStart()`.  However, I don't think we should be doing this development in `jetty-10`,  if we're doing it, it should be in `jetty-12`.

Reply:
This was a possible solution for what was brought up in Issue #11274, which meant it should be done in Jetty 10 thru Jetty 12.

But the OP (@vmassol) made some good points (see https://github.com/jetty/jetty.project/issues/11266#issuecomment-1892611178) and has convinced me that the approach in this PR isn't a great solution.

I'm totally OK with this being only for Jetty 12.
Want me to recreate this for Jetty 12 only? or drop it as a concept?

Reply:
IMHO we should do it for 12, and also update the doco for it with the new context attributes.

Reply:
Will do, and I'll add unit tests!